### PR TITLE
Create fuel_emission_category table

### PR DESCRIPTION
**Pull request by BCerki**
Card: https://github.com/bcgov/cas-ciip-portal/issues/2145

To check I'd correctly entered the data:
- checked that the number of inserted records (216) matched the sum of records in the spreadsheet (added up all the 1s)
- made sure that the breakdown of each category (eg 8 flaring) matched the amount of times find/replace found that category

### CODE REVIEW

File: schema/deploy/tables/fuel_emission_category.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+-- Grant ciip_industry_user permissions
+perform ggircs_portal_private.grant_permissions('select', 'fuel_emission_category', 'ciip_industry_user');
+
+
+
+end
+$grant$;
+
+-- Enable row-level security
+-- alter table ggircs_portal.incremental_fuel_charge_baseline enable row level security;
```
Comment (BCerki) [inline schema/deploy/tables/fuel_emission_category.sql:41]: The card doesn't ask for this but it was on the other tables so we might need it here too? (but with the correct table name)

Comment (dleard) [reply schema/deploy/tables/fuel_emission_category.sql:41]: For this table we don't need any granular RLS policies. The grants you've defined here cover it

**Review by dleard — APPROVED** [review]
This looks great. Awesome work :100: 
I manually checked the values in the table against the data we received from CAS & all values are correct.
This branch just needs a rebase

**Review by dleard — APPROVED** [review]
This looks great. Awesome work. 
I manually checked the values in the table against the data we received from CAS & all values are correct.

**Pull request merged**